src/api/handlers/supported_language.py

```diff
@@ -18,23 +16,22 @@ def registerRoutes(self):
         self.add("/supported_languages.create", self.create)
         self.add("/supported_languages.add", self.create)
         self.add("/supported_languages.list", self.list)
-        self.add("/supported_languages.disable", self.disable)
-        self.add("/supported_languages.enable", self.enable)
 
     def info(self, request):
         context: Context = request.context
         args: dict = context.args
 
-        # verify the body of the incoming request
-        self.validator.expect("id", str, is_required=True)
-        self.validator.rename("supported_language_id", "id")
-        args, err = self.validator.verify(args, strict=True)
-        if err:
-            return err
+        code = args.get('code', None)
```

Comment:
`code` is too generic. 
```suggestion
        code = args.get('language_code', None)
```